### PR TITLE
[WFCORE-2939]:  Disabled deployments in server group enabled after WAR is updated

### PR DESCRIPTION
**Pull request by ehsavoie**
Adding enabled flag to be able to distinguish between disabled, enabled or leave as it is.

Jira: https://issues.jboss.org/browse/WFCORE-2939

### CODE REVIEW

File: cli/src/main/java/org/jboss/as/cli/handlers/DeployHandler.java [hunk 1]
```diff
@@ -563,14 +586,27 @@ public ModelNode buildRequestWithoutHeaders(CommandContext ctx) throws CommandFo
 
         final boolean force = this.force.isPresent(args);
         final boolean disabled = this.disabled.isPresent(args);
+        final boolean enabled = this.enabled.isPresent(args);
         final String serverGroups = this.serverGroups.getValue(args);
         final boolean allServerGroups = this.allServerGroups.isPresent(args);
         final boolean archive = isCliArchive(f);
 
+        if (enabled && disabled) {
+            throw new CommandFormatException("enabled and disabled can't be used at the <");
```
Comment (jfdenise) [inline cli/src/main/java/org/jboss/as/cli/handlers/DeployHandler.java]: Message seems incomplete

File: cli/src/main/resources/help/deploy.txt [hunk 2]
```diff
@@ -72,6 +72,12 @@ ARGUMENTS
                     and the existing deployment is not disabled, the existing
                     deployment will be undeployed.
 
+ --enabled       - indicates that the deployment has to be added to the
```
Comment (jfdenise) [inline cli/src/main/resources/help/deploy.txt]: A missing whitespace

**Comment by ehsavoie** [comment]
retest this please

**Comment by ehsavoie** [comment]
retest this please

**Pull request merged**